### ISSUE TITLE
Deployment to Railway

### ISSUE THREAD

**Issue by BlackDahlia313** [directus-labs/guest-authoring#8]
### What is your idea?

Create a guest blog that showcases how to quickly deploy Directus onto the Railway platform.

### What are the key takeaways from your post?

- Guide users how to deploy Directus to Railway
- Show how to connect a managed Postgres DB
- Show how to connect a managed Redis DB
- Connect a persistent volume or point user to an S3 compatible service as volumes has [constraints](https://docs.railway.app/reference/volumes). 

### What type of post is this?

Self-Hosting Guide

### Country of residence

US

### Terms & Conditions

- [X] You understand that submitting this form does not mean we accept your proposal. We will accept proposals at our discretion.
- [X] You understand that to be paid you will need to generate an invoice to us once the final draft of your work is completed.
- [X] You understand that you will grant us an unlimited license to post and promote your work as part of this program. You may repost, but using our copy as the canonical source.
- [X] You are aged 18 or over.

**Comment by phazonoverload** [comment]
Yup, as discussed I'm up for this. Same rate as the DO one - $350. Cheers!

**Comment by phazonoverload** [comment]
Hey @BlackDahlia313 it's been a while since this one first came up and was approved. Can I get a indication of if you're still working on it or whether you're no longer interested? 

If you're up for it - can I get an indication of when you might have a first draft done?

**Comment by phazonoverload** [comment]
Just wanted to share that we've recently updated our process for guest authoring to have deadlines for first drafts so we can better plan content. 

The deadline for first draft is February 2 😄 In future, we'll agree this ahead of time, but it's aligned what what we have agreed off-platform.

**Comment by BlackDahlia313** [comment]
> Just wanted to share that we've recently updated our process for guest authoring to have deadlines for first drafts so we can better plan content.
> 
> The deadline for first draft is February 2 😄 In future, we'll agree this ahead of time, but it's aligned what what we have agreed off-platform.

Sounds good. I'll knock this out before than. Sorry for the delay.

**Comment by BlackDahlia313** [comment]
@phazonoverload Same issue as DO with 10.8.3 fyi. Ill have the draft to you tomorrow or sunday.

Thanks!

**Comment by phazonoverload** [comment]
Hey @BlackDahlia313 I'm feeling like this one might not pan out? It's reaching a point where I'm ok with that, but need to know one way or another if I should keep this in the plan for scheduling and budgeting. Can we get clarity on that by the end of Friday?

**Comment by phazonoverload** [comment]
Going to remove this one off the roster for now as it's been dangling for a while. If you feel like doing it at a later point, just open a new proposal issue 😄 

All good by the way - I know it's been a busy time for you.
